applications/ULFapplication/python_scripts/ulf_strategy_PGLASS.py

```diff
@@ -37,9 +37,9 @@ def __init__(self, model_part, time_scheme, linear_solver, convergence_criteria,
         self.pDx = self.space_utils.CreateEmptyVectorPointer()
         self.pb = self.space_utils.CreateEmptyVectorPointer()
```

Comment:
Strategies should be in cpp

Reply:
@loumalouomega I think we are having some sort of conceptual misunderstanding here. We are not planning to change the structure of the entire ULFApplication in this pull request (when the application was originally added we were not asked to reproduce the format of say FluidDynamicsApplication or other "user-oriented" standardized application). All the python solvers in ULF are divided into two parts: solver and strategy. We are neither interested in switching to JSON at the moment as we are using a different problem type.